### PR TITLE
🐛 Fix webserver twilio env

### PR DESCRIPTION
**Pull request by YuryHrytsuk**
## What do these changes do?
🐛 Add missing TWILIO ENV to a `webserver`. These ENV variables are necessary for some products for login functionality. Otherwise, webserver fails to start because the [validation](https://github.com/ITISFoundation/osparc-simcore/blob/2e33ecd5b08b1f7dc2cf2a3f2cbcf98a875484ca/services/web/server/src/simcore_service_webserver/login/settings.py#L59C1-L66) does not pass

## Related issue/s
* https://github.com/ITISFoundation/osparc-simcore/pull/4491

## How to test
* Run whole ops stack locally and ensure that `webserver` launches succesfully

### CODE REVIEW

**Review by GitHK — APPROVED** [review]

**Review by sanderegg — APPROVED** [review]

**Review by mrnicegyu11 — APPROVED** [review]
perfect, thanks a lot

**Review by sanderegg — COMMENTED** [review]
looking good. As discussed I am not sure whether this can have consequences on local builds. I guess the e2e tests would complain if there is one.

**Review by sanderegg — APPROVED** [review]

File: packages/settings-library/tests/test_twilio.py [hunk 1]
```diff
@@ -12,10 +10,6 @@
 def test_twilio_settings_within_envdevel(
     mock_env_devel_environment: dict[str, str], monkeypatch: MonkeyPatch
 ):
-    # in .env-devel these are for the oment undefined
-    with pytest.raises(ValidationError):
-        TwilioSettings.create_from_envs()
```
Comment (GitHK) [inline packages/settings-library/tests/test_twilio.py:17]: If you are removing this, please create a test without the fixture that automatically loads the `.env-devel` (I think it comes in via `mock_env_devel_environment`) and ensure it raises an error.

**Review by GitHK — CHANGES_REQUESTED** [review]

File: packages/settings-library/tests/test_twilio.py (hunk 1 above)
Comment (GitHK) [reply packages/settings-library/tests/test_twilio.py:17]: ignore this

**Pull request merged**